app/schemas/models/course.schema.coffee

```diff
@@ -4,16 +4,19 @@ CourseSchema = c.object {title: 'Course', required: ['name']}
 c.extendNamedProperties CourseSchema  # name first
 
 _.extend CourseSchema.properties,
+  i18n: {type: 'object', title: 'i18n', format: 'i18n', props: ['name', 'description' ]}
   campaignID: c.objectId()
   concepts: c.array {title: 'Programming Concepts', uniqueItems: true}, c.concept
   description: {type: 'string'}
   duration: {type: 'number', description: 'Approximate hours of content'}
   pricePerSeat: {type: 'number', description: 'Price per seat in USD cents.'} # deprecated
   free: { type: 'boolean' }
-  screenshot: c.url {title: 'URL', description: 'Link to course screenshot.'}
+  screenshot: { type: 'string', title: 'URL', description: 'Link to course screenshot.'}
```

Comment:
I might add a `c.path` and allow it to be either, or something like that. Not super important, but has data validation niceness.